packages/danger/src/PullRequestManager.ts

```diff
@@ -0,0 +1,174 @@
+   * {
+   *   [DEFAULT_CHANGELOG_ENTRY_KEY]: <title of this pr without tags>
+   * }
+   * Otherwise, it tries to parse PR's body.
+   */
+  parseChangelogSuggestionFromDescription(): ChangelogEntries {
+    const changelogEntries = {
+      [DEFAULT_CHANGELOG_ENTRY_KEY]: {
+        type: DEFAULT_ENTRY_TYPE,
+        message: this.pullRequest.title.replace(/\[.*\]/, '').trim(),
+      },
+    };
+
+    const parseLine: (line: string) => void = line => {
+      const tags = this.parseTagsFromLine(line);
+      if (!tags) {
+        warn(`Couldn't parse line: ${line}.`);
+        return;
+      }
+
+      changelogEntries[tags.packageName] = {
+        type: tags.type,
+        message: line.replace(/\[.*\]/, '').trim(),
+      };
+    };
+
+    parseLine(this.pullRequest.title);
+
+    const changelogSection = this.pullRequest.body.match(/#\schangelog(([^#]*?)\s?)*/i)?.[0];
+    if (changelogSection) {
+      changelogSection
+        .replace(/^-/, '')
+        .split('\n')
+        .slice(1)
+        .map(line => line.trim())
+        .filter(line => line.length > 0)
+        .forEach(parseLine);
+    }
+
+    return changelogEntries;
+  }
+
+  async createOrUpdatePRAsync(
+    missingEntries: { packageName: string; content: string }[]
+  ): Promise<PullRequest | null> {
+    const dangerHeadRef = `@danger/add-missing-changelog-to-${this.pullRequest.number}`;
+    const dangerBaseRef = this.pullRequest.head.ref;
+
+    const fileMap = missingEntries.reduce(
+      (prev, current) => ({
+        ...prev,
+        [getPackageChangelogRelativePath(current.packageName)]: current.content,
+      }),
+      {}
+    );
+
+    await this.githubApi.createOrUpdateBranchFromFileMap(fileMap, {
+      baseBranchName: dangerBaseRef,
+      branchName: dangerHeadRef,
+      message: `${dangerTags} ${dangerMessage}`,
+    });
+
+    const prs = await this.githubApi.getOpenPRs({
+      fromBranch: dangerHeadRef,
+      toBranch: dangerBaseRef,
+    });
+
+    if (prs.length > 1) {
+      warn("Couldn't find the correct pull request. Too many open ones.");
+      return null;
+    }
+
+    if (prs.length === 1) {
+      return prs[0];
+    }
+
+    return this.githubApi.openPR({
+      fromBranch: dangerHeadRef,
+      toBranch: dangerBaseRef,
+      title: `${dangerTags} ${dangerMessage} to #${this.pullRequest.number}`,
+      body: `${dangerMessage} to #${this.pullRequest.number}`,
+    });
+  }
+
+  private parseTagsFromLine(
+    line: string
+  ): { packageName: string | typeof DEFAULT_CHANGELOG_ENTRY_KEY; type: ChangelogEntryType } | null {
+    const result: {
+      packageName: string | typeof DEFAULT_CHANGELOG_ENTRY_KEY;
+      type: ChangelogEntryType;
+    } = {
+      type: DEFAULT_ENTRY_TYPE,
+      packageName: DEFAULT_CHANGELOG_ENTRY_KEY,
+    };
+
+    const tags = line.match(/\[[^\]]*\]/g);
+    if (!tags) {
+      return result;
+    }
+
+    for (const tag of tags) {
+      const entryType = parseEntryType(tag);
+      if (entryType) {
```

Comment:
`parseEntryType` can return `ChangelogEntryType.BUG_FIXES` which equals to `0` which is falsy, so it'll be treated as a package name 😄